From 07a5c4732f1c28ffcb873ee04500e3cb813c50b4 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Tue, 7 Dec 2021 22:28:04 +0100
Subject: [PATCH] Remove ramdac

ramdac drivers have been removed from xserver since version 21.0.99.1
and
https://gitlab.freedesktop.org/xorg/xserver/-/commit/f0385fb420158ac3bc1c4c325431ffc5c62344bb
resulting in the following build failure:

In file included from trident_bank.c:37:
trident.h:41:10: fatal error: xf86RamDac.h: No such file or directory
   41 | #include "xf86RamDac.h"
      |          ^~~~~~~~~~~~~~

Fixes:
 - http://autobuild.buildroot.org/results/c81ac8075af257e8626d9d097270be7a7b4a1496

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status:
https://gitlab.freedesktop.org/xorg/driver/xf86-video-trident/-/merge_requests/1]
---
 src/trident.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/trident.h b/src/trident.h
index 5cadf52..c82de4c 100644
--- a/src/trident.h
+++ b/src/trident.h
@@ -38,7 +38,6 @@
 #include "xaa.h"
 #endif
 #include "xf86fbman.h"
-#include "xf86RamDac.h"
 #include "compiler.h"
 #include "vgaHW.h"
 #include "xf86i2c.h"
@@ -103,7 +102,6 @@ typedef struct {
     int			useEXA;
     int			Chipset;
     int			DACtype;
-    int			RamDac;
     int                 ChipRev;
     int			HwBpp;
     int			BppShift;
@@ -169,7 +167,6 @@ typedef struct {
     CARD32		BltScanDirection;
     CARD32		DrawFlag;
     CARD16		LinePattern;
-    RamDacRecPtr	RamDacRec;
     int			CursorOffset;
     xf86CursorInfoPtr	CursorInfoRec;
     xf86Int10InfoPtr	Int10;
-- 
2.33.0

